internal/output/html/style.html

```diff
@@ -396,18 +400,27 @@
   }
 
   .severity-short p {
-    border-radius: 1px;
     text-align: center;
   }
 
+  .severity-short-first p {
+    border-top-left-radius: 2px;
```

Comment:
nit: Make border radius 4px for everything to match osv.dev

Reply:
I set a 4px border-radius for most elements (all tags), except for the filter container, which has an 8px radius. I think it looks better this way (also osv.dev has different border-radius setting for different components).